Tests/UnitTests/Core/Seeding/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
+set(unittest_extra_libraries ActsExamplesFramework)
```

Comment:
As discussed: This should not depend on the Examples if it is a Core unit test

Reply:
Yes, I changes the location of the test!